docker/gwc/overrides/cache/geowebcache.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This looks like it's the same as the geowebcache.oem.xml file above. If so, do we need both?